book/src/SUMMARY.md

```diff
@@ -1,61 +1,61 @@
 # Summary
 
-- [👋 Welcome](./1.welcome/README.md)
-  - [Why an SDK?](./1.welcome/1.need-for-sdk.md)
-  - [Packages](./1.welcome/2.packages.md)
-- [🚀 Getting Started](./2.getting-started/README.md)
-  - [Quickstart](./2.getting-started/1.quickstart.md)
-  - [Anatomy of an App](./2.getting-started/2.code-basics.md)
-  - [Running in Teams](./2.getting-started/3.running-in-teams.md)
-- [Essentials](./3.essentials/README.md)
-  - [App Basics](./3.essentials/1.app-class.md)
-  - [Listening To Events](./3.essentials/1.on-event.md)
-  - [Listening To Activities](./3.essentials/2.on-activity.md)
-    - [Activity Reference](./3.essentials/activity-ref.md)
-  - [Sending Messages](./3.essentials/4.sending-messages.md)
-    - [Proactive Messages](./3.essentials/5.proactive-messaging.md)
-  - [Using The API Client](./3.essentials/6.api.md)
-- [Teams](./4.teams/README.md)
-  - [Core Concepts](./4.teams/1.core-concepts.md)
-  - [Teams Manifest]()
+- [👋 Welcome](./welcome/README.md)
+  - [Why an SDK?](./welcome/need-for-sdk.md)
+  - [Packages](./welcome/packages.md)
+- [🚀 Getting Started](./getting-started/README.md)
+  - [Quickstart](./getting-started/quickstart.md)
+  - [Anatomy of an App](./getting-started/code-basics.md)
+  - [Running in Teams](./getting-started/running-in-teams.md)
+- [Essentials](./essentials/README.md)
+  - [App Basics](./essentials/app-class.md)
+  - [Listening To Events](./essentials/on-event.md)
+  - [Listening To Activities](./essentials/on-activity.md)
+    - [Activity Reference](./essentials/activity-ref.md)
+  - [Sending Messages](./essentials/sending-messages.md)
+    - [Proactive Messages](./essentials/proactive-messaging.md)
+  - [Using The API Client](./essentials/api.md)
+- [Teams](./teams/README.md)
+  - [Core Concepts](./teams/core-concepts.md)
+  - [Teams Manifest](./teams/manifest.md)
```

Comment:
@corinagum ok to put in?

Reply:
Thanks for the catch - forgot to do it :/